src/pages/myAdmin/releaseNotes.tsx

```diff
@@ -0,0 +1,868 @@
+      </li>
+    </ul>
+    <h2>EditUserContact</h2>
+    <ul>
+      <li>
+        In an upcoming release, we will update the <strong>EditUserContact</strong> method logic to validate provided addresses, and provide the option to override invalid addresses, if the user has the <code className="small-code-sample">Contact-Override</code> role.
+      </li>
+    </ul>
+  </div>
+);
+
+const updateNovember142022: ReactNode = (
+  <div className="paragraph">
+    <h2>ApiOnlineOrder</h2>
+    <ul>
+      <li>
+        A new source parameter has been added to the <code className="small-code-sample">ApiOnlineOrder</code> object: <code className="small-code-sample">OrderSource.</code>
+      </li>
+      <li>
+        The Order source parameter allows you to determine the channel used to place the order: MyAdmin, Marketplace, or API.
+      </li>
+    </ul>
+    <h2>MyInstallLogs</h2>
+    <ul>
+      <li>
+        A new <strong>GetMyInstallLogs</strong> method has been added to allow you to easily retrieve your installation record information.
+      </li>
+    </ul>
+    <h2>CreateExternalVendorDeviceBulkAsync</h2>
+    <ul>
+      <li>
+        A new method, <strong>CreateExternalVendorDeviceBulkAsync</strong>, has been added to make it easier for Marketplace Partners to offer products that do not connect to Geotab gateways.
+      </li>
+      <li>
+        The <strong>CreateExternalVendorDeviceBulkAsync</strong> method allows Marketplace Partners to register devices using serial numbers, as well as the option to include IMEI and SIM numbers, so they show up in MyAdmin.
+      </li>
+      <li>
+        To use this method, you must have the <code className="small-code-sample">DeviceRegister-Bulk</code> role.
+      </li>
+    </ul>
+  </div>
+);
+
+const updateSeptember192022: ReactNode = (
+  <div className="paragraph">
+    <h2>ApiDeviceContract</h2>
+    <ul>
+      <li>
+        <strong>ApiDeviceContract</strong> has been updated with a boolean value for the <code className="small-code-sample">isAutoActivated</code> parameter to indicate whether a device was auto-activated by Geotab using True or False.
+      </li>
+      <li>
+        The True label indicates the device was auto-activated by Geotab.
+      </li>
+    </ul>
+    <h2>AccountsFromSerialsAsync</h2>
+    <ul>
+      <li>
+        A new API, <strong>GetAccountsFromSerialsAsync</strong>, has been added to allow our Partners with multiple accounts identify the account associated with a specified device, and facilitates the use of other MyAdmin APIs that require account information.
+      </li>
+    </ul>
+    <h2>OnlineOrderStatus</h2>
+    <ul>
+      <li>
+        A new parameter, <code className="small-code-sample">poExactMatch</code>, has been added to <strong>GetOnlineOrderStatus</strong> to allow youto search for orders that match a specific PO number.
```

Comment:
```suggestion
        A new parameter, <code className="small-code-sample">poExactMatch</code>, has been added to <strong>GetOnlineOrderStatus</strong> to allow you to search for orders that match a specific PO number.
```
Space between 'you' & 'to'